src/jaxsim/physics/model/physics_model.py

```diff
@@ -99,11 +99,14 @@ def __post_init__(self):
             self._link_spatial_inertias = jnp.stack(
                 [self._link_inertias_dict[l.index] for l in ordered_links]
             )
-
-            s_min = jnp.hstack([j.position_limit[0] for j in ordered_joints])
-            s_max = jnp.hstack([j.position_limit[1] for j in ordered_joints])
-            self._joint_position_limits_min = jnp.vstack([s_min, s_max]).min(axis=0)
-            self._joint_position_limits_max = jnp.vstack([s_min, s_max]).max(axis=0)
+            if len(ordered_joints) > 0:
+                s_min = jnp.hstack([j.position_limit[0] for j in ordered_joints])
+                s_max = jnp.hstack([j.position_limit[1] for j in ordered_joints])
+                self._joint_position_limits_min = jnp.vstack([s_min, s_max]).min(axis=0)
+                self._joint_position_limits_max = jnp.vstack([s_min, s_max]).max(axis=0)
+            else:
+                self._joint_position_limits_min = []
+                self._joint_position_limits_max = []
```

Comment:
Thanks a lot @traversaro! I did some tests and I believe that we can avoid the use of `jnp.hstack` as for a list it would provide the same output of `jnp.array`. In this way we can skip the check on the length

```suggestion
                s_min = jnp.array([j.position_limit[0] for j in ordered_joints])
                s_max = jnp.array([j.position_limit[1] for j in ordered_joints])
                self._joint_position_limits_min = jnp.vstack([s_min, s_max]).min(axis=0)
                self._joint_position_limits_max = jnp.vstack([s_min, s_max]).max(axis=0)
```

Reply:
Thanks, with the minor fix https://github.com/ami-iit/jaxsim/pull/104/commits/69bd58698ed31e876d1013757f89e91ae1cc22e5 it is indeed working fine, I will squash the history to propose this.